searchcore/src/vespa/searchcore/proton/server/proton.cpp

```diff
@@ -219,7 +232,6 @@ Proton::Proton(const config::ConfigUri & configUri,
       _threadPool(128 * 1024),
       _distributionKey(-1),
       _isInitializing(true),
-      _isReplayDone(false),
```

Comment:
Why is this removed?

Reply:
It was not used. It was set, but never used.